Update webhook docs for issues (#9930)
getsentry/sentry-docs: docs/product/integrations/integration-platform/webhooks/issues.mdx

### action

- type: string

> - description: can be `created`, `resolved`, `assigned`, `archived` or `unresolved`

Reviewer: ```suggestion
- description: can be `created`, `resolved`, `assigned`, `archived`, or `unresolved`
```

## Payload

Payload includes additional information about the issue. Some important fields to pay attention to are:

### status

> Some issue webhooks are getting triggered on status change of an issue. An issue status can be `resolved`, `unresolved` or `ignored` (archived).

Reviewer: ```suggestion
Some issue webhooks are triggered on status change of an issue. An issue status can be `resolved`, `unresolved`, or `ignored` (archived).
```

### substatus

> Substatus of an issue is giving more details about a status, specially if one status can mean multiple things. For example, an issue status can change to `unresolved` because it is now `escalating` or because a resolved issue is `regressed`. Possible substatuses are: `archived_until_escalating`, `archived_until_condition_met`, `archived_forever`, `escalating`, `ongoing`, `regressed` and `new`.

Reviewer: ```suggestion
A status can mean multiple things, so substatus gives more details about the status. For example, an issue status can change to `unresolved` because it is now `escalating` or because a resolved issue is `regressed`. Possible substatuses are: `archived_until_escalating`, `archived_until_condition_met`, `archived_forever`, `escalating`, `ongoing`, `regressed` and `new`.
```

## Payload

Payload includes additional information about the issue. Some important fields to pay attention to are:

### status
Some issue webhooks are triggered on status change of an issue. An issue status can be `resolved`, `unresolved`, or `ignored` (archived).

### substatus
A status can mean multiple things, so substatus gives more details about the status. For example, an issue status can change to `unresolved` because it is now `escalating` or because a resolved issue is `regressed`. Possible substatuses are: `archived_until_escalating`, `archived_until_condition_met`, `archived_forever`, `escalating`, `ongoing`, `regressed` and `new`.

### statusDetails

> `statusDetails` includes more detailed information about the action being done on the issue, for example is the issue is being resolved or archived. These details are:

Reviewer: ```suggestion
`statusDetails` includes more detailed information about `resolved` and `archived` status changes on an issue. These details can include:
```

> * `inRelease`: The release that the resolution is in for example `latest`
> * `inNextRelease`: `True` if the resolution will be in the next release
> * `inCommit`: Includes information about the resolution commit, for example `repository` and `commit`
> * `ignoreDuration`: Duration for an archived issue before it's turns to escalated in minutes

Reviewer: ```suggestion
* `inRelease`: The release that the resolution is in (for example, `latest`)
* `inNextRelease`: `True` if the resolution will be in the next release
* `inCommit`: Includes information about the resolution commit (such as `repository` and `commit`)
* `ignoreDuration`: The duration (in minutes) that an archived issue should be ignored before it gets escalated.
```

can you add more details about the behavior or ignoreDuration? For example, will a new occurrence still escalate the issue even if the duration hasn't been met? Or will a new occurrence trigger the duration countdown? If the issue is resolved during the ignoreduration, then what happens? Does it always escalate once the duration is up?

> * `ignoreCount`: Maximum number of occurences in `ignoreWindow` minutes for an archived issue before it gets escalated                                 
> * `ignoreUserCount`: Maximun number of users in `ignoreUserWindow` minutes who are affected by an archived issue before it gets escalated

Reviewer: Where do `ignoreWindow` and `ignoreUserWindow` come from? Are they also part of statusDetails? Can we add them to this list (before ignoreCount and ignoreUserCount)? 

```suggestion
* `ignoreCount`: Maximum number of occurrences that can happen within the duration of `ignoreWindow` for an archived issue before it gets escalated                                 
* `ignoreUserCount`: Maximum number of users who can be affected by an archived issue within the duration of `ignoreUserWindow` before it gets escalated
```

Author: Yes, they're part of the same map. My thinking was because these don't make sense alone I should document them together. I'll add them
